blog/2023-07-18-wing-cloud-launch.md

```diff
@@ -0,0 +1,281 @@
+
+### Reducing iteration time from minutes to milliseconds
+
+The #1 factor in developer productivity is the time it takes to perform a single iteration
+(sometimes called the *inner dev loop* cycle). This is the time it takes to run my code and check
+that it does what I expected it to do. 
+
+The more an application leans on the cloud, the more this cycle is broken. If for every iteration, I
+need to deploy my code to a personal cloud account, it means that a cycle takes multiple minutes
+(depending on the resources I deploy). Then, having to switch to some other system to run my code
+and trace the logs also increases this iteration time and breaks my development flow.
+
+Some tools are trying to address this problem by "hot swapping" runtime code in the cloud, or by
+proxying requests into my local machine so I can debug locally. But those solutions are only partial as they only take care of the runtime code and not on the entire application. If something changes in my infrastructure, I go back to waiting.
+
+I personally believe that in most cases using real cloud resources for development iterations is
+never going to get us to developer experience nirvana. This is because when we provision resources
+in the cloud, we leverage their [control plane](https://en.wikipedia.org/wiki/Control_plane), which,
+by definition, is not optimized for latency.
+
+In Wing, thanks to the cloud abstraction layer, we are able to take a different approach. The same mechanism we have for cloud portability is used to implement a local cloud simulation target for each resource in the WCL. This means that my entire Wing application can run in a local machine and functionally tested.
+
+### Tests as first-class citizens
+
+We love tests. We think tests are likely the most important asset a software team builds. Our code
+can be changed and refactored as much as we like, but our tests capture the expectations from our
+system.
+
+The reality of the cloud today is that it is almost impossible to write effective tests that cover
+more than just the application code. With Wing, this can now change.
+
+The fact we have a simulator means that it is now possible to **write unit tests** which not only
+validate my runtime code, but also how this code is interacting with the infrastructure around it
+and with other pieces of my system.
+
+To that end, the Wing language, the SDK and the Wing Console all work together to offer first-class
+support for [cloud testing](https://www.winglang.io/docs/concepts/tests).
+
+<!--![screenshot]()-->
+
+Tests in Wing are multi-cloud by default and can all run in parallel because they use cloud
+functions as their compute resource. Every test you write can be execute locally in the cloud
+simulator and can also be deployed and executed across all supported cloud providers.
+
+We believe that being able to run and test cloud applications locally is a game changer for
+developers. It's definitely a game changer for us!
```

Comment:
This paragraph is on the weaker side to me

Reply:
ok removed it. not sure why you didn't like it but whateverrrrrr